manifests/pxeboot/kickstart.ks

```diff
@@ -172,6 +172,26 @@ chmod +x /etc/yum.repos.d/marvell-tools-beaker.sh
 
 /etc/yum.repos.d/marvell-tools-beaker.sh @__YUM_REPO_URL__@ @__YUM_REPO_ENABLED__@
 
+EXTRA_PACKAGES=( @__EXTRA_PACKAGES__@ )
+if [ "@__DEFAULT_EXTRA_PACKAGES__@" = 1 ] ; then
+    case "$(sed -n 's/^VERSION_ID="\(.*\)"/\1/p' /etc/os-release)" in
+        9.4)
+            # The rhel-9.4 kernel currently lacks patches for the Marvell DPU. Install
+            # a rebuild of upstream netnet kernel (with a rhel-9.4 config).
+            EXTRA_PACKAGES+=(
+                "http://file.brq.redhat.com/~thaller/marvell-octeon-10-tools/kernel-6.13.0_rc1.octeontx2_1_3d64c3d3c6d8+-1/kernel-6.13.0_rc1.octeontx2_1_3d64c3d3c6d8+-1.aarch64.rpm"
```

Comment:
I expect kernel-modules and/or kernel-module-extras rpm. Does this mean that the marvell kernel driver is baked in?

Reply:
The module is still a separate module. But the way the kernel is build (in the commit message), they are all in one RPM. That's because this is `make srcrpm-pkg` from upstream, which unlike the RHEL variants don't split the package.

Reply:
I see, got it thanks!